app/views/bounty_mailer/open_bounty.html.erb

```diff
@@ -0,0 +1,6 @@
+<p><strong>The open bounties for this week:</strong></p>
+<% @bounties.each do |bounty| %>
+  <h3><%= bounty.title %></h3>
+  <p>Amount: <%= bounty.amount %>$</p>
+  <p><%= link_to "View the source code", bounty.url %></p>
+<% end %>
```

Comment:
This part is a little odd, looking at the Rails Guides, the example mailer view has the Doctype and HTML tags in it, however, I'm pretty confident that's not required unless we're overriding the markup that's in `app/views/layouts/mailer.html.erb`

If you take a look at the  `app/views/layouts/mailer.html.erb` layout, everything we add in this file will be rendered with `<%= yield %>` tag in between the body tags